packages/usa-file-input/src/usa-file-input.twig

```diff
@@ -22,12 +22,11 @@
     class="usa-file-input"
     type="file"
     name="{{ id }}"
-    {% if hint or accept or multiple or disabled_state -%}
-    {% if hint -%}aria-describedby="{{ aria_hint }}"{%- endif -%}
-    {% if accept -%}accept="{{ accept }}"{%- endif -%}
-    {% if multiple -%} multiple{%- endif -%}
-    {% if disabled_state == 'disabled' %} disabled{%- endif %}
-    {% if disabled_state == "aria-disabled" -%} aria-disabled="true"{%- endif -%}
-    {% endif %}
+    {% if hint %} aria-describedby="{{ aria_hint }}"{% endif %}
+    {% if accept %} accept="{{ accept }}"{% endif %}
+    {% if multiple %} multiple{% endif %}
+    {% if disabled_state == 'disabled' %} disabled{% endif %}
+    {% if disabled_state == "aria-disabled" %} aria-disabled="true"{% endif %}
+    {% if accepted_files_message %} data-errorMessage="{{ accepted_files_message }}"{% endif %}
```

Comment:
Are the extra spaces needed if we've removed the tags to strip whitespace?

```suggestion
    {% if hint %}aria-describedby="{{ aria_hint }}"{% endif %}
    {% if accept %}accept="{{ accept }}"{% endif %}
    {% if multiple %}multiple{% endif %}
    {% if disabled_state == 'disabled' %}disabled{% endif %}
    {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}
    {% if accepted_files_message %}data-errorMessage="{{ accepted_files_message }}"{% endif %}
```

Reply:
Thank you for flagging this! I missed that I could remove the spaces now that I've removed the strip whitespace flag!

Resolved in [39e16e6](https://github.com/uswds/uswds/pull/6168/commits/39e16e6942b8e761aeadd3ae05baa56ea0991068)

Reply:
Thank you! 

Ran the build HTML script [`npm run build:html`]. No regressions found in templates that we use on site.